constellations/gemini-admin/src/index.js

```diff
@@ -10,32 +10,44 @@ import { ProgramPage } from "./programs/program.js";
 import { ProgramEditPage } from "./programs/programEdit.js";
 import { ClassPage } from "./classes/class.js";
 import { ClassEditPage } from "./classes/classEdit.js";
-import { AchievePage } from "./achieve/achieve.js";
-import { AchieveEditPage } from "./achieve/achieveEdit.js";
+import { SessionPage } from "./session/session.js";
+import { SessionEditPage } from "./session/sessionEdit.js";
 import { AnnouncePage } from "./announce/announce.js";
 import { AnnounceEditPage } from "./announce/announceEdit.js";
+import { AchievePage } from "./achieve/achieve.js";
+import { AchieveEditPage } from "./achieve/achieveEdit.js";
 import { LocationPage } from "./location/location.js";
 import { LocationEditPage } from "./location/locationEdit.js";
 import { SemesterPage } from "./semester/semester.js";
 import { SemesterEditPage } from "./semester/semesterEdit.js";
 
-const Achieve = () => <AchievePage />;
-const AchieveEdit = () => <AchieveEditPage />;
-const AchieveEditMatch = ({ match }) => (
-    <AchieveEditPage Id={match.params.Id} />
-);
-const Announce = () => <AnnouncePage />;
-const AnnounceEdit = () => <AnnounceEditPage />;
-const AnnounceEditMatch = ({ match }) => (
-    <AnnounceEditPage announceId={match.params.announceId} />
-);
```

Comment:
There isn't anything you have to change, but I'm just wondering why you chose to put the files in this particular order?

Reply:
I just put them in the same order as the tabs.